NOnion/Crypto/Kdf/Kdf.fs

```diff
@@ -49,3 +55,52 @@ module Kdf =
             ForwardKey = forwardKey
             KeyHandshake = keyHandshake
         }
+
+    let computeRfc5869Kdf (ikm: array<byte>) : KdfResult =
```

Comment:
this func is not nested so should be PascalCase

Reply:
the previous function is already using non-pascalcase naming, inconsitant naming is worse

Reply:
what previous function? where's the inconsistency here? I don't understand

Reply:
Check the original file, that same module has another function with the same naming scheme

Reply:
We have the same Contain asser used in 2 other tests, so we have to either change all of them or let this one slide as well also Contains is not an extension method

Reply:
so you mean I forgot to tell you in the PR that introduced this function that it should be PascalCase? then let's fix it now in an additional commit

Reply:
Fixed